fix: add extra steps for monitoring docs (#2379)
codenvy/che-docs: modules/administration-guide/partials/proc_collecting-dev-workspace-operator-metrics-with-prometheus.adoc

.Prerequisites

> * The `devworkspace-controller-metrics` Service is exposing metrics on port `8443`. This is pre-configured by default.

Reviewer: I recommend dropping the dash because that's the mainstream variant according to https://www.merriam-webster.com/dictionary/pre-configure:
```suggestion
* The `devworkspace-controller-metrics` Service is exposing metrics on port `8443`. This is preconfigured by default.
```

> * The `devworkspace-webhookserver` Service is exposing metrics on port `9443`. This is pre-configured by default.

Reviewer: Same as above.
```suggestion
* The `devworkspace-webhookserver` Service is exposing metrics on port `9443`. This is preconfigured by default.
```

. Configure Prometheus to scrape metrics from port `8443` exposed by the `devworkspace-controller-metrics` Service and from port `9443` exposed by the `devworkspace-webhookserver` Service.
+

> NOTE: The example monitoring stack already creates the `prometheus-config` ConfigMap with an empty configuration. Edit the `data` field of the ConfigMap to provide the Prometheus configuration details.

Reviewer: ```suggestion
NOTE: The example monitoring stack already creates the `prometheus-config` ConfigMap with an empty configuration. To provide the Prometheus configuration details, edit the `data` field of the ConfigMap.
```

          tls_config:
            insecure_skip_verify: true
          static_configs:
            - targets: ['devworkspace-webhookserver.__<DWO_{orch-namespace}>__:9443'] <5>
----
<1> The rate at which a target is scraped.
<2> The rate at which the recording and alerting rules are re-checked.
<3> The resources that Prometheus monitors. In the default configuration, two jobs, `DevWorkspace` and `DevWorkspace webhooks`, scrape the time series data exposed by the `devworkspace-controller-metrics` and `devworkspace-webhookserver` Services.

> <4> Scrape metrics from port `8443`. Replace `__<DWO_namespace>__` with the {orch-namespace} where the `devworkspace-controller-metrics` `Service` is located.

Reviewer: I recommend keeping `The` to make it clear that it's not used as a verb (in the imperative form) here and to align it with the previous lines that all start with `The`.

The same for the next callout below.

Author: Updated to:
```
<4> The scrape target for the metrics from port `8443`. Replace `__<DWO_namespace>__` with the {orch-namespace} where the `devworkspace-controller-metrics` `Service` is located.
<5> The scrape target for the metrics from port `9443`. Replace `__<DWO_namespace>__` with the {orch-namespace} where the `devworkspace-webhookserver` `Service` is located.
```

How does this look?

Reviewer: That looks great imo.

David, you're welcome to resolve this thread.

<5> Scrape metrics from port `9443`. Replace `__<DWO_namespace>__` with the {orch-namespace} where the `devworkspace-webhookserver` `Service` is located.
====

> . Scale down and scale up the `Prometheus` Deployment to read the updated ConfigMap from the previous step.

Reviewer: If this makes sense:
```suggestion
. Scale the `Prometheus` Deployment down and up to read the updated ConfigMap from the previous step.
```

+
[subs="+attributes,quotes"]
----
$ {orch-cli} scale --replicas=0 deployment/prometheus -n monitoring && {orch-cli} scale --replicas=1 deployment/prometheus -n monitoring
----

> .Verification steps

Reviewer: Same as above, no need for `steps` after `.Verification`.

. Use port forwarding to access the `Prometheus` Service locally:
+
[subs="+attributes,quotes"]
----
$ {orch-cli} port-forward svc/prometheus 9090:9090 -n monitoring
----
. Verify that all targets are up by viewing the targets endpoint at `localhost:9090/targets`.
. Use the Prometheus console to view and query metrics:
** View the metrics at `localhost:9090/metrics`.

> ** Query metrics from `localhost:9090/graph`.

Reviewer: If this makes sense, consider having `the` in both bullet points or in neither.

Author: In my latest commit, took the same approach as https://github.com/eclipse-che/che-docs/pull/2379#discussion_r905344015

> ** The metrics are available at `localhost:9090/metrics`.
> ** The metrics can be queried from `localhost:9090/graph`.

Reviewer: Just a last-minute suggestion:
```suggestion
** View metrics at `localhost:9090/metrics`.
** Query metrics from `localhost:9090/graph`.
```
